src/ads/BaseAd.tsx

```diff
@@ -115,8 +115,7 @@ export const BaseAd = React.forwardRef<GoogleMobileAdsBannerView, GAMBannerAdPro
 
     const style = sizes.includes(BannerAdSize.FLUID)
       ? {
-          width: '100%',
-          height: dimensions[1],
+          flex: 1
```

Comment:
@caiqueex this PR may be interesting for you if you want to take a look and help review

Reply:
Hi Mike, I o the test with this PR code. the result was it:

<img width="361" alt="Captura de Tela 2022-12-10 às 23 26 20" src="https://user-images.githubusercontent.com/8368079/206883576-e7f37450-210a-4261-9abb-723d1cedf30a.png">

I dont know why,  but it has that black background color and content to the right side

Reply:
<img width="277" alt="Captura de Tela 2022-12-10 às 23 31 22" src="https://user-images.githubusercontent.com/8368079/206883668-9ba28f0d-9ff7-4906-9833-43dd8342d647.png">

@mikehardy

Reply:
TestId.BANNER has 320x50. Does the black area represent 320+? Could you make a native ad on GAM and give it a 100% width and fixed height?